app/javascript/src/components/Common/DeleteAlert.jsx

```diff
@@ -0,0 +1,21 @@
+// @ts-nocheck
```

Comment:
Remove this. In VSCode the errors will be there. Don't mind it.

Reply:
Any proper solution to remove those errors as there are many of them ?